packages/editor/src/components/browser/TypeBrowser.tsx

```diff
@@ -137,7 +207,7 @@ const TypeBrowser = (props: { value: string; onChange: (value: string) => void;
           }
         }}
       >
-        {mainFilter.length > 0 ? (
+        {(mainFilter.length > 0 && allSearchActive) || !allSearchActive ? (

```

Comment:
instead of this duplication of logic, what should be shown if the allsearch is active or not, can you just show this empty message if the rowmodel is empty? Or does this not work? 
And of course then the message would be a little bit different, more like 'No type found, use the search for ...'